tests/test_eth_benchmark.py

```diff
@@ -0,0 +1,65 @@
+import shutil
+import struct
+import tempfile
+import unittest
+import os
+
+from manticore.core.plugin import Plugin
+from manticore.core.smtlib import ConstraintSet, operators
+from manticore.core.smtlib.expression import BitVec
+from manticore.core.smtlib import solver
+from manticore.core.state import State
+from manticore.ethereum import ManticoreEVM, DetectInvalid, DetectIntegerOverflow, Detector, NoAliveStates, ABI, EthereumError
+from manticore.platforms.evm import EVMWorld, ConcretizeStack, concretized_args, Return, Stop
+from manticore.core.smtlib.visitors import pretty_print, translate_to_smtlib, simplify, to_constant
+
+import shutil
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+# FIXME(mark): Remove these two lines when logging works for ManticoreEVM
+from manticore.utils.log import init_logging, set_verbosity
+init_logging()
+set_verbosity(0)
+
+class EthBenchmark(unittest.TestCase):
+    """ https://consensys.net/diligence/evm-analyzer-benchmark-suite/ """
+    def setUp(self):
+        self.mevm = ManticoreEVM()
+        self.mevm.verbosity(0)
+        self.worksp = self.mevm.workspace
+
+    def tearDown(self):
+        self.mevm=None
+        shutil.rmtree(self.worksp)
+
+    def _test_assert(self, name, should_find):
+        """
+        Tests DetectInvalid over the consensys benchmark suit
+        """
+        mevm = self.mevm
+
+        d = DetectInvalid()
+        mevm.register_detector(d)
+
+        filename = os.path.join(THIS_DIR, 'binaries', 'benchmark', '{}.sol'.format(name))
```

Comment:
An interesting approach to static paths like this: https://github.com/pwndbg/pwndbg/blob/dev/tests/binaries/__init__.py

Then it can be used as: https://github.com/pwndbg/pwndbg/blob/dev/tests/test_loads.py#L63-L64